presto-main/src/main/java/com/facebook/presto/util/CompilerUtils.java

```diff
@@ -54,12 +60,20 @@ public static ParameterizedType makeClassName(String baseName)
 
     public static <T> Class<? extends T> defineClass(ClassDefinition classDefinition, Class<T> superType, Map<Long, MethodHandle> callSiteBindings, ClassLoader parentClassLoader)
     {
-        return defineClass(classDefinition, superType, new DynamicClassLoader(parentClassLoader, callSiteBindings));
+        try {
```

Comment:
This is not needed if you already handles the exception in the other method?

Reply:
both were public so I just added it :) now removed.